doc/Language/objects.pod6

```diff
@@ -760,22 +760,25 @@ say "y: ", $p.y;
 C<Mu.new> calls method L<bless|/routine/bless> on its invocant, passing all
 the named L<arguments|/language/functions#Arguments>. C<bless> creates the new
 object, and then walks all subclasses in reverse method resolution order
-(i.e. from L<Mu|/type/Mu> to most derived classes) and in each class checks for
-the existence of a method named C<BUILD>. If the method exists, the
-method is called with all the named arguments from the C<new> method. If
-not, the public attributes from this class are initialized from named
-arguments of the same name. In either case, if neither C<BUILD> nor the
-default mechanism has initialized the attribute, default values are
-applied. This means that C<BUILD> may change an attribute, but it does
-not have access to the contents of the attribute declared as its
-default; these are available only during C<TWEAK> (see below), which can
-'see' the contents of an attribute initialized in the declaration of the
-class.
-
-X<|TWEAK>
-After the C<BUILD> methods have been called for a class, methods named
-C<TWEAK> are called for that class, if they exist, again with all the
-named arguments that were passed to C<new>. See an example of its use below.
+(i.e. from L<Mu|/type/Mu> to most derived classes). In each class the following
+steps take place in the order given here:
+
+=item It checks for the existence of a method named C<BUILD>. If the method
+exists, the method is called with all the named arguments from the C<new>
+method.
+
+=item If no C<BUILD> method was found, the public attributes from this class
+are initialized from named arguments of the same name.
+
+=begin item
+All attributes that have not been touched in any of the previous steps have
+their default values applied:
+
```

Comment:
I now added the anchor a few lines down. I've never used `X<|STUFF>` before. I hope I did it right.

Reply:
> The first file is OK, except if you want to address the two or three quirks. However, deleting a big chunk of documentation without a very good reason (like it being totally wrong, in which case it would be better to fix it) is something we don't really like.
> What I would suggest is to leave the first file in this PR (with changes, if you want) and open an issue to discuss options about the second one. It's true that all that in Types/Mu might not seem ideal, but in that case, it would be better to move it somewhere else.

I'd like to keep the changes together, as I tried to add the information I removed in `doc/type/Mu.pod6` to `doc/Language/objects.pod6` (Mostly the bit about the danger of using `new`). Separating the removal and addition part hides this a bit.
I'm not in a hurry with this PR, so I'm fine discussing the changes before merging. Is this OK with you?